### PR TITLE
chore: add test status markdown file

### PR DESCRIPTION
**Pull request by underfin**
### Description

Add test status markdown file and link it at README.MD

### Test Plan
-

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 Fast JavaScript/TypeScript bundler in Rust with Rollup-compatible API.
 
+# Rollup test status
+
+![status](https://github.com/rolldown/packages/rollup-tests/test/status.svg)
```
Comment (hyf0) [inline README.md]:
```suggestion
![status](./packages/rollup-tests/test/status.svg)
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 Fast JavaScript/TypeScript bundler in Rust with Rollup-compatible API.
 
+# Rollup test status
+
+![status](./packages/rollup-tests/test/status.svg)
```
Comment (hyf0) [inline README.md]:
```suggestion
![status](./packages/rollup-tests/src/status.svg)
```

**Review by hyf0 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 Fast JavaScript/TypeScript bundler in Rust with Rollup-compatible API.
 
+# Rollup test status
+
+![status](./packages/rollup-tests/src/status.svg)
+
+It is status for rollup function/file-hashes directory test cases.
```
Comment (hyf0) [inline README.md]: I thought this is status of all tests of rollup. Why emphasize `function/file-hashes`?

Comment (underfin) [reply README.md]: Because other test case look like need diffrent format, we don't support it at now.

File: README.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 Fast JavaScript/TypeScript bundler in Rust with Rollup-compatible API.
 
+# Rollup test status
+
+See [rollup test status](./packages/rollup-tests/src/status.md).
+
+It is status for rollup function and file-hashes directory test cases.
```
Comment (hyf0) [inline README.md]:
```suggestion
It shows how many tests from rollup are passed by rolldown. The tests currently include the `function` and `file-hashes` directories of rollup tests.
```

**Review by hyf0 — APPROVED** [review]

**Pull request merged**